### PR TITLE
Docs dir renaming

### PR DESCRIPTION
**Pull request by calvellido**
I am a big fan of consistency too, but to be able to publish through GitHub pages we must have the documentation under the `docs` directory.

Please have a look to see if this renaming has had any undesirable effect beyond the occurences I've found along the project.

### CODE REVIEW

**Comment by 47degdev** [comment]
***Hood benchmark comparison:***
:heavy_check_mark: `Decoding` (Threshold: 180.50806940585198)

Benchmark | Value
|--|--|
master_benchmark | 11273.57761149028
helios_benchmark | 11493.921467834367


:warning: `Parsing` (Threshold: 464.3722646574531)

Benchmark | Value
|--|--|
master_benchmark | 22275.294139199403
helios_benchmark | 22047.214515686755

File: settings.gradle [hunk 1]
```diff
@@ -8,5 +8,5 @@ include ':helios-benchmarks'
 include ':helios-sample'
 include ':helios-optics'
 include ':helios-dsl-syntax-compiler'
-include 'helios-docs'
+include 'docs'
```
Comment (AdrianRaFo) [inline settings.gradle]: Could we move this to the top or last position just for esthetics reasons

Comment (calvellido) [reply settings.gradle]: Ha!  Yeah, sure :nerd_face:

**Comment by AdrianRaFo** [comment]
@calvellido I just realized the microsite icon is the Arrow one could we solve that here?

**Comment by calvellido** [comment]
Woops, yeah, let me create a proper one

**Comment by calvellido** [comment]
Actually... It is your caché @AdrianRaFo

**Pull request merged**